src/activities/webhooks/Elsa.Webhooks.Persistence.YesSql/Documents/WebhookDefinitionDocument.cs

```diff
@@ -4,7 +4,7 @@ namespace Elsa.Webhooks.Persistence.YesSql.Documents
 {
     public class WebhookDefinitionDocument : YesSqlDocument
     {
-        public string WebhookDefinitionId { get; set; } = default!;
+        public string WebhookId { get; set; } = default!;
```

Comment:
Let's rename this to `DefinitionId` to be consistent with the name used in `WorkflowDefinitionDocument`?

Reply:
WebhookId renamed to DefinitionId